scripts/deploy.sh

```diff
@@ -85,6 +85,10 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
     sed -i "s/aio1/$(hostname)/" /etc/openstack_deploy/openstack_user_config.yml
     sed -i "s/aio1/$(hostname)/" /etc/openstack_deploy/conf.d/*.yml
   fi
+  if [ "$DEPLOY_CEILOMETER" != "yes" ]; then
+    rm -f /etc/openstack_deploy/conf.d/ceilometer.yml
+    sed -i 's/swift_ceilometer_enabled:.*/swift_ceilometer_enabled: false/' /etc/openstack_deploy/user_variables.yml
```

Comment:
The current default seems to be set to false, better to be explicit, but if that line is instead commented out we don't actually disable it (since we won't remove the comment).  If we only want to change the variable if it's defined then I suggest starting the search with a `^`.  Also, not a global replace, but that's likely intentional.  Also, also to match the rest of the file it should be `False` not `false`.